spec/controllers/applications_controller_spec.rb

```diff
@@ -57,7 +57,7 @@
         expect(response).to render_template :show
       end
 
-      it "should not render application of another user" do
+      it "should redirect to root for another user's application" do
```

Comment:
Updated the name of this example to match https://github.com/doubleunion/arooo/blob/6c170311d8d7ac11a8c3f4041035cdf9eb95d589/spec/controllers/applications_controller_spec.rb#L96 -- same assertion for a different page.